corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/session_nav.js

```diff
@@ -39,6 +39,13 @@ FormplayerFrontend.module("SessionNavigate", function (SessionNavigate, Formplay
         },
     };
 
+    FormplayerFrontend.on("apps:currentApp", function () {
+        var oldRoute = Backbone.history.getFragment();
+        var appId = Util.getAppId(oldRoute);
```

Comment:
do we have no reference to the appId saved anywhere? i.e. we have to parse it from the url?